src/components/InventoryV2/Search.tsx

```diff
@@ -98,6 +142,45 @@ export default function Search(props: SearchProps): JSX.Element | null {
           />
         </Box>
         <InventoryFilters filters={filters} setFilters={setFilters} origin={origin} />
+        {origin === 'Batches' && (
+          <Box sx={{ marginTop: theme.spacing(0.5) }}>
+            <Tooltip title={strings.FILTER}>
+              <Button
+                id='filterSpecies'
+                onClick={(event) => event && handleFilterClick(event)}
+                type='passive'
+                priority='ghost'
+                icon='filter'
+              />
+            </Tooltip>
+            <Popover
+              id='simple-popover'
+              open={Boolean(filterAnchorEl)}
+              anchorEl={filterAnchorEl}
+              onClose={handleFilterClose}
+              anchorOrigin={{
+                vertical: 'bottom',
+                horizontal: 'center',
+              }}
+              transformOrigin={{
+                vertical: 'top',
+                horizontal: 'center',
+              }}
+              className={classes.popoverContainer}
+            >
+              <FilterGroup
+                initialFilters={filterGroupFilters}
+                fields={filterGroupColumns}
+                onConfirm={(_filterGroupFilters: Record<string, SearchNodePayload>) => {
+                  handleFilterClose();
+                  setFilterGroupFilters(_filterGroupFilters);
+                  setFilters({ ...filters, ...convertFilterGroupToMap(_filterGroupFilters) });
+                }}
+                onCancel={handleFilterClose}
+              />
+            </Popover>
+          </Box>
+        )}
```

Comment:
At what point should we consider merging this with https://github.com/terraware/terraware-web/blob/main/src/components/InventoryV2/view/InventorySeedlingsTable.tsx#L391? I suspect we will add more filters eventually, but maybe _that_ is when it makes sense to combine?

Reply:
You're probably right that next time we touch this filter would make sense. But to be honest I could have broke it into its own component here. I can take a step back and do it now if you like. What do you think?

Reply:
Hard to say, I think I am indifferent if it is only written twice. I don't want to be overly forceful  with DRY... I think if we have to write it a third time, or if we have to duplicate a ticket in the future because it has to be implemented in both places, then we should think about it. I'm OK to move on for now

Reply:
Okay, I'm with you there. Sounds good